starter/js/atm.js

```diff
@@ -1,43 +1,51 @@
-//Begin with the document ready function
-
-    //Checking account deposit function
-
-      //On click of the depositChecking button
-
-        //Get value from the amountChecking input field
-
-        //Take that value and add it to the existing value in the checkingBalance div
-
-    //Checking account withdrawl funtion
-
-      //On click of the withdrawChecking button
-
-        //Get value from the amountChecking input field
-
-        // If that value is greater than the value in the account ignore that action
-        // In other words if this would put the account into a negative balance do not allow it
-
-        //Else subtract that value from the current amount in the checking account
-
-    //Savings account deposit function
-
-      //On click of the depositSavings button
-
-        //Get value from the amountSavings input field
-
-        //Take that value and add it to the existing value in the savingsBalance div
-
-    //Savings account withdraw funtion
-
-      //On click of the withdrawl button
-
-        //Get value from the amountSavings input field
-
-         //If that value is greater than the value in the account ignore that action
-         //In other words if this would put the account into a negative balance do not allow it
-
-         //Else subtract that value from the current amount in the savings account
-
-// Bonus-- get the two accounts to work with each other and allow for overdraft protection
-
-// Double Bonus-- This isnt very DRY.  Using the keyword "this" see if you can make the withdraw and deposit functions work for both accounts
+$(document).ready(function () {
+
+  $('#depositChecking').on('click', function() {
+    var currentBalance = parseInt($('#checkingBalance').text());
+    var value = parseInt($('#amountChecking').val());
+    var newBalance = currentBalance + value;
+    $('#checkingBalance').text(newBalance);
+    $('#amountChecking').val('');
+    if (newBalance > 0) {
+      $('#check').css('background-color','#E3E3E3');
+    }
+  });
+
+  $('#withdrawChecking').on('click', function() {
+    var currentBalance = parseInt($('#checkingBalance').text());
+    var value = parseInt($('#amountChecking').val());
+    var newBalance = currentBalance - value;
+    $('#checkingBalance').text(newBalance);
+    $('#amountChecking').val('');
+    if (newBalance < 0) {
+      $('#checkingBalance').text(currentBalance);
+      alert('Insufficient Funds')
+    }
+    if (newBalance = 0) {
```

Comment:
This won't work. When testing equality, use `===`. You're doing assignment with only one `=`